### ISSUE TITLE
Model from API

### ISSUE THREAD

**Issue by christopherquenneville** [oobabooga/text-generation-webui#5083]
Hi,

Thank you for creating this great web UI.
It's been great so far to run models locally on my machine.

I'm wondering if I could use this as an interface for API requests instead of running models locally.
OpenAI APIs could be an example, but I'm specifically looking at Azure AI Studio integrations.

I know you have done this: https://github.com/oobabooga/text-generation-webui/wiki/12-%E2%80%90-OpenAI-API#examples
Basically looking for the opposite.

**Comment by yhyu13** [comment]
This is an interesting idea, I am also wonder if we make textgen webui as an endpoint for sending/receiving messages from chatgpt, and enjoying all extensions from textgen webui all the way.

**Comment by StableLlama** [comment]
This is the same as #5627 - so we do have many people asking for the same but sadly the @github-actions bot closes these away and nobody sees that this request was sent in multiple times
